proto/streamlit/proto/NumberInput.proto

```diff
@@ -32,7 +32,7 @@ message NumberInput {
   bool has_max = 12;
 
   DataType data_type = 13;
-  double default = 14;
+  double value = 14;
```

Comment:
Like Button.proto comment: To change the protobuf, choose a new numerical value for "value" and explicitly deprecate the word "default" as well as its number.